setup.py

```diff
@@ -35,7 +35,7 @@ def find_version(fname):
 history = open('HISTORY.rst').read().replace('.. :changelog:', '')
 requirements = [
     str(requirement.req)
-    for requirement in parse_requirements('requirements.txt')
+    for requirement in parse_requirements('requirements.txt', session=uuid.uuid1())
```

Comment:
Actually, all I wanted was a unique id to fill in the session variable. It is not used afterwards anyways. but yes pip.download.PipSession() would also work there.